Jenkinsfile

```diff
@@ -33,7 +34,6 @@ pipeline {
                 python3 -m venv $TMPDIR
                 . $TMPDIR/bin/activate
                 pip install tox  # for tox supporting --parallel--safe-build
```

Comment:
Side note: `--parallel-safe-build` is marked as deprecated in the docs. After a quick search it's not clear to me if there's a direct replacement for it.